src/components/AppBar.tsx

```diff
@@ -42,6 +46,10 @@ const isLoading = (apiStatus: APIStatusType): boolean => {
 const AppBar: React.FC<Props> = ({ user, userAPIStatus, handleLogin, handleLogout }): ReactElement => {
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
 
+  const dispatch = useAppDispatch();
+  const appTheme = useAppSelector((state: any) => state.preference.appTheme);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/batnoter/batnoter/pullRequest?prid=9609391)
```suggestion
  const appTheme = useAppSelector((state: unknown) => state.preference.appTheme);
```